src/main/java/core/basesyntax/model/Figure.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.model;
+
+public abstract class Figure implements Drawable, AreaCalculable {
+    protected String color;
+
+    @Override
+    public void draw() {
+        System.out.println(this);
+    }
```

Comment:
not required in abstract class, let children classes implement this method in their own way